lib/percy/common/version.rb

```diff
@@ -1,5 +1,5 @@
 module Percy
   module Common
-    VERSION = '1.1.0'
+    VERSION = '1.1.0'.freeze
```

Comment:
Comment: We should consider adopting `# frozen_string_literal: true`, which would avoid the need for this call.